fiat-api/src/main/java/com/netflix/spinnaker/fiat/api/Permissions.java

```diff
@@ -79,13 +77,8 @@ public boolean isRestricted() {
     return this.permissions.values().stream().anyMatch(groups -> !groups.isEmpty());
   }
 
-  public boolean isAuthorized(Set<Role> userRoles) {
-    return !getAuthorizations(userRoles).isEmpty();
-  }
-
-  public Set<Authorization> getAuthorizations(Set<Role> userRoles) {
```

Comment:
`Permission` used to depend on `Role` which in turn brought in a bunch of stuff that I didn't want to pull into `fiat-api`.

Reply:
Looking at `Role`, it's not immediately obvious to me what extra things would be brought alongside?